src/components/v5/common/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useCallback, useMemo, useState } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+import debounce from 'lodash.debounce';
+
+import FormError from '~v5/shared/FormError';
+
+import { TextareaProps } from './types';
+
+const displayName = 'v5.common.Textarea';
+
+const DEFAULT_MAX_CHAR_NUMBER = 90;
+
+const Textarea: FC<TextareaProps> = ({
+  textareaTitle = '',
+  maxCharNumber = DEFAULT_MAX_CHAR_NUMBER,
+  placeholder = '',
+  showFieldLimit,
+}) => {
+  const { formatMessage } = useIntl();
+  const [currentCharNumber, setCurrentCharNumber] = useState(0);
+  const [isTyping, setIsTyping] = useState(false);
+
+  const isError = useMemo(
+    () => currentCharNumber > maxCharNumber,
+    [currentCharNumber, maxCharNumber],
+  );
+
+  const onChangeTyping = useCallback(() => setIsTyping(false), []);
+
+  const debounced = useMemo(
+    () => debounce(onChangeTyping, 1000),
+    [onChangeTyping],
+  );
+
+  const onChange: React.ChangeEventHandler<HTMLTextAreaElement> = useCallback(
+    (e) => {
+      setIsTyping(true);
+      const { value } = e.target;
+      setCurrentCharNumber(value.length);
+      debounced(value);
+    },
+    [debounced],
+  );
+
+  return (
+    <div className="flex flex-col gap-1">
+      {textareaTitle && (
+        <label htmlFor="message" className="text-gray-700 text-md font-medium">
```

Comment:
`text-md font-medium` is the same as `text-1`.
Please, check the `main.global.css` before using font styles, there should be placed most of the them.